core/pom.xml

```diff
@@ -103,6 +103,10 @@
       <groupId>org.apache.zookeeper</groupId>
       <artifactId>zookeeper</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.gaul</groupId>
+      <artifactId>modernizer-maven-annotations</artifactId>
```

Comment:
Can this be an optional dependency?

Reply:
Yes, it just needs added to the server-base, gc, master, tserver and test pom files.